python-sdk/tutorials/automl-with-azureml/forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -397,7 +377,7 @@
     "    freq=\"D\",  # Set the forecast frequency to be daily\n",
     ")\n",
     "\n",
-    "# We will disable the enable_early_stopping flag to ensure the DNN model is recommended for demonstration purpose.\n",
+    "# We only allow TCNForecaster for this notebook to only recommend the TCN model.\n",
```

Comment:
Please consider changing this to: "We only allow TCNForecaster we set the `alowed_models` parameter to reflect this."